tuxemon/core/states/dialog/__init__.py

```diff
@@ -60,3 +63,29 @@ def next_text(self):
             return text
         except IndexError:
             return None
+
+    def draw_avatar(self):
+        if not self.avatar:
+            return
+
+        # If the prefix is "monster" the second parameter refers to a monster
+        # If this parameter is a number, we're referring to a monster slot in the party
+        # If this parameter is a string, we're referring to a monster name
+        # If there's no prefix, the parameter represents the path to a sprite file
+        params = self.avatar.split(" ")
+        if params[0] == "monster":
+            if params[1].isdigit():
+                player = self.game.player1
+                slot = int(params[1])
+                avatar_sprite = player.monsters[slot].get_sprite("menu")
+            else:
+                avatar_monster = monster.Monster()
+                avatar_monster.load_from_db(params[1])
+                avatar_monster.flairs = {} # Don't use random flair graphics
```

Comment:
i see why you added this line to prevent flairs.  please open a follow up ticket to organize the flairs into groups according to the sprite type they overlay, such as front, back, menu, etc

Reply:
ideally, if we have flairs, they should also be on the menu sprite (like hair ribbons, etc) and hiding them or not drawing them would be completely automatic.

Reply:
This is only an exception since we don't want random flairs when referencing a monster by name (that's not an actual monster in the world). I'll keep that in mind for later... though I'd rather not make flairs more complex, they are grouped in a simple but efficient way currently.